glitchdotcom/Glitch-Community
# Pricing page new design

## Links
* [pricing-page-new-design.glitch.me](https://pricing-page-new-design.glitch.me)
* [[ch7328]](https://app.clubhouse.io/glitch/story/7328/update-logged-in-pricing-page-with-new-designs): Update logged in pricing page
* [Figma mockups](https://www.figma.com/file/ES2qByV3V8qc8Ep961HQyQ/121719_Pufferfish_Design_EM?node-id=1026%3A104)

## GIF/Screenshots:
<div>
<img width="210" alt="Screen Shot 2020-01-29 at 2 40 12 PM" src="https://user-images.githubusercontent.com/7584833/73391908-d55f0500-42a6-11ea-8815-24b0146b7c7f.png">
<img width="300" alt="Screen Shot 2020-01-29 at 2 40 04 PM" src="https://user-images.githubusercontent.com/7584833/73391973-f0317980-42a6-11ea-8470-1d174041eb7d.png">
<img width="514" alt="Screen Shot 2020-01-29 at 2 39 56 PM" src="https://user-images.githubusercontent.com/7584833/73392090-253dcc00-42a7-11ea-9ca7-e0758b218702.png">
</div>

<table>
<thead>
<th>Logged in, not subscribed</th>
<th>Logged in, subscribed</th>
<th>Not logged in (once feature flag is removed)</th>
</thead>
<tr>
<td>
<img width="220" alt="Screen Shot 2020-01-29 at 2 38 46 PM" src="https://user-images.githubusercontent.com/7584833/73392472-e4928280-42a7-11ea-88e2-88c5cde6247f.png">
</td>
<td>
<img width="470" alt="Screen Shot 2020-01-29 at 2 38 26 PM" src="https://user-images.githubusercontent.com/7584833/73392473-e4928280-42a7-11ea-8f26-f44d051c4f90.png">
</td>
<td>
<img width="390" alt="Screen Shot 2020-01-29 at 2 37 05 PM" src="https://user-images.githubusercontent.com/7584833/73392471-e4928280-42a7-11ea-9f77-f27a7d98ed1a.png">
</td>
</table>

## Changes:
This updates /pricing to use Edwin's mockups.

*This PR anticipates that eventually this page will be visible to users who are not logged in. Right now this page is behind a feature flag (which requires login), so the "Sign In to Subscribe" button will never be visible*

## How To Test:
* Enable pufferfish on /secret
* Go to /pricing, you should see this page
* Sign up for a paid account, the button content should now read "Manage Your Subscription"
